### DISCUSSION TITLE
duplicated sponsoring messages on feed list

### DISCUSSION THREAD

**Discussion by oe** [community/community#36202] [Other Feature Feedback, Questions, & Ideas]
Microsoft is sponsoring a lot projects,  that's great! But why these same recipients keep showing on my feed list from time to time?

![CleanShot 2022-10-15 at 10 34 55](https://user-images.githubusercontent.com/1655294/195965083-910d388c-9199-459c-a676-78336ad24cdb.png)

And can you guys merge the same category messages from same user into one? just like this:
![CleanShot 2022-10-15 at 10 44 57](https://user-images.githubusercontent.com/1655294/195965399-81a56bc1-0fc3-4a91-8c66-119f30c9c0eb.png)

**Comment by ossanna16** [comment]
Hey there @oe and thank you for asking a good question! I'm glad you found our community 🙂
